src/linear_algebra/matrix/nonsingular_inverse.lean

```diff
@@ -103,6 +104,33 @@ begin
     apply h }
 end
 
+@[simp] lemma cramer_one : cramer (1 : matrix n n α) = 1 :=
+begin
+  ext i j,
+  simp only [linear_map.one_apply, function.comp_app, linear_map.coe_comp, linear_map.coe_single],
+  have : (1 : matrix n n α) i = pi.single i 1,
+  { ext k,
+    by_cases hk : k = i,
+    { subst hk,
+      simp },
+    { simp [matrix.one_apply, hk, pi.single_eq_of_ne, ne.symm hk] } },
+  rw [←transpose_one, ←this, cramer_transpose_row_self],
+  simp [matrix.one_apply],
+end
```

Comment:
Ideally this should be like two lines: `convert cramer_transpose_row_self; simp`. I'll see if I can extract some lemmas to make this a two-liner.

Reply:
Check out commit 292216d1df

Reply:
Thanks!